src/components/TopNav/NavMenu.js

```diff
@@ -24,6 +24,7 @@ export const NavMenu = ({ dispatch, history, ticketIds }) => {
     <MenuItem key='logout' primaryText={<Link to='/' onClick={() => dispatch(auth.logOut)}>LogOut</Link>} />
     { ticketIds && ticketIds.map(transformIdToTicketLink) }
     <MenuItem key='debug' primaryText={<Link to='/debug' >Debug</Link>} />
+    <MenuItem key='load uncorrelated events' primaryText={<Link to='/events'>Load Events</Link>} />
```

Comment:
For a new end-user, this menu option may be a bit confusing. Something along the lines of 'Events For All Incidents' would clarify the difference between this page and the page for specific incident timelines.